### DISCUSSION TITLE
trigger use gesture on an area around the object

### DISCUSSION THREAD

**Discussion by tilnea** [pmndrs/react-spring#1220] [Support]
Hi,

I'm using React use Gesture and React Three Fiber and I want to increase the area around an object (in this case a line) where the user can drag the line. Right now the user need to be right on the line do drag it, and I wan to increase the area a little bit around the line to give users better UX. Do you know how I can do this?

source code - https://codesandbox.io/s/drag-circle-w0i6z?file=/src/index.js

Thanks!

**Comment by mrgoonie** [comment]
I don't think this question is related in this repo, should it be asked in React Three Fiber repo instead? 

However, I'm thinking to an approach where you create another line with the same shape & position, but bigger line width, and opacity at 0.001 😆

**Reply by tilnea** [reply]
Ok, I imagine it is in react-use-gesture, but there was no discussion in there so that's why I added my question here.

In React Three Fiber I can increase my threshold, but react use gesture does not seem to take that to account. So I want to change the threshold react-use-gesture uses to set that an object is the target object.

But thanks for your answer anyway! I appreciate you taking the time to answer my question :)
